include/git2/status.h

```diff
@@ -311,6 +311,24 @@ GIT_EXTERN(int) git_status_list_new(
 	git_repository *repo,
 	const git_status_options *opts);
 
+/**
+ * Gather file status information compared to a specified tree.
+ *
+ * Similar to `git_status_list_new`, but instead of using HEAD it allows
+ * another target tree to be specified.
+ *
+ * @param out Pointer to store the status results in
+ * @param repo Repository object
+ * @param opts Status options structure
+ * @param head Tree to compare the index to
```

Comment:
Please specify that `head` can actually be `NULL` (behaving as an empty iterator then).